Monitor/stats - Copy - Copy - Copy.py

```diff
@@ -0,0 +1,15 @@
+import psutil
```

Comment:
Copy-Copy-Copy-Copy ?

Reply:
Ignore the file names, finally only a single copy of this file exists in each VM.